src/commands/new.rs

```diff
@@ -352,11 +353,25 @@ fn path_safe(text: &str) -> PathBuf {
 }
 
 fn validate_name(name: &str) -> Result<String, String> {
-    if NAME.is_match(name) {
-        Ok(name.to_owned())
-    } else {
-        Err("Name must start with a letter".to_owned())
+    let splits = name.split(|c| !char::is_alphanumeric(c));
+    let invalid_split_displays = splits
+        .filter(|s| !NAME.is_match(s))
```

Comment:
Do we need the regex? Wouldn't `filter(|s| s.starts_with(char::is_ascii_alphabetic))` work?